themes/custom/apigee_kickstart/includes/node.inc

```diff
@@ -28,7 +28,7 @@ function apigee_kickstart_preprocess_node(&$variables) {
     ], $variables);
   }
 
-  $apigee_api_content_types = ['apidoc', 'graphql_doc', 'asyncapi_doc'];
+  $apigee_api_content_types = ['apidoc', 'graphql_doc', 'asyncapi_doc', 'freeform_doc'];
```

Comment:
Should this section be a configuration ?
When someone adds a new content type. They will need to come here and make changes.

Reply:
This helps in displaying the `API Style` name on the API listing page, which comes in the right bottom corner.
![Screenshot 2022-09-19 8 15 26 PM](https://user-images.githubusercontent.com/75600200/191045910-b0122827-3acd-4646-ba3e-1cca0cd70244.png)

We can create a `API Style` setting page and there we can set the value for each content type.